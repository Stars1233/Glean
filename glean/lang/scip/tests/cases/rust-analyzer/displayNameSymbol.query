query: scip.DisplayNameSymbol { displayName = "Prefix" }
perf: True
transform: [gensort, []]
