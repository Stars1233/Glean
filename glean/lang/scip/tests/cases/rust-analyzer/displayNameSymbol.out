[
  "@generated",
  {
    "key": {
      "displayName": { "key": "Prefix" },
      "symbol": { "key": "rust-analyzer cargo relative_path 0.0.0 Prefix#" }
    }
  }
]